src/test/java/uk/gov/register/presentation/functional/testSupport/DBSupport.java

```diff
@@ -13,19 +11,20 @@
 import java.sql.SQLException;
 import java.time.Instant;
 import java.util.List;
-import java.util.SortedMap;
-import java.util.TreeMap;
+import java.util.concurrent.atomic.AtomicInteger;
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
+import static uk.gov.register.presentation.functional.TestEntry.anEntry;
+
 public class DBSupport {
     public static void main(String[] args) throws IOException, SQLException {
         String filePath = args[1];
 
         DBSupport dbSupport = new DBSupport(TestDAO.get(args[2], "postgres"));
         try (Stream<String> lines = Files.lines(new File(filePath).toPath(), Charset.defaultCharset())) {
-            List<String> collect = lines.collect(Collectors.toList());
-            dbSupport.publishMessages(args[0], collect);
+            AtomicInteger index = new AtomicInteger(0);
```

Comment:
Using `map` method on a list stream shouldn't be a problem as it keeps the ordering (if I understood your concern correctly).

This is used in `main` method to load the data for development environment. An entry number  can be provided with in the data file instead of using `AtomicInteger` iff needed.